nextflow_schema.json

```diff
@@ -559,7 +595,14 @@
                     "description": "Method used to save pipeline results to output directory.",
                     "help_text": "The Nextflow `publishDir` option specifies which intermediate files should be saved to the output directory. This option tells the pipeline what method should be used to move these files. See [Nextflow docs](https://www.nextflow.io/docs/latest/process.html#publishdir) for details.",
                     "fa_icon": "fas fa-copy",
-                    "enum": ["symlink", "rellink", "link", "copy", "copyNoFollow", "move"],
+                    "enum": [
+                        "symlink",
+                        "rellink",
+                        "link",
+                        "copy",
+                        "copyNoFollow",
+                        "move"
```

Comment:
```suggestion
                        "copy",
                        "copyNoFollow",
                        "link",
                        "move",
                        "rellink",
                        "symlink"
```

Reply:
This comes from the pipeline template, so best would be to fix it there, as otherwise I'm gonna have conflicts here next time 😄